src/components/sidebar/right-sidebar/datepicker/DateSlider.tsx

```diff
@@ -0,0 +1,97 @@
+import React, {useCallback, useEffect, useRef, useState} from 'react';
+import './DateSlider.css';
+
+interface DSprops {
+    min: number;
+    max: number;
+    values: Date[];
+    minDate: Date;
+    onChange: Function;
+}
+
+export default function DateSlider(props: DSprops){
+
+    const {min, max, minDate} = props;
+    const [values, setValues] = useState<number[]>([(props.values[0].getTime() - minDate.getTime())/86400000,
```

Comment:
is there a better way to do this? if not can we at least assign 86400000 to a variable with a descriptive name (say secondsPerYear or something) for better readability?